### DISCUSSION TITLE
Welcome to PuLID Discussions!

### DISCUSSION THREAD

**Discussion by ToTheBeginning** [ToTheBeginning/PuLID#8] [Announcements]
## 👋 Welcome!
  We’re using Discussions as a place to connect with other members of our community. We hope that you:
  * Ask questions you’re wondering about.
  * Share ideas.
  * Engage with other community members.
  * Welcome others and are open-minded. Remember that this is a community we
  build together 💪.

  To get started, comment below with an introduction of yourself and tell us about what you do with this community.

**Comment by spacegrey23** [comment]
Hi everyone, for me pulid is primarily a creative tool that allows you to create portraits, combine with styles or input photos themselves. I'm very happy that such a project was created, because it allows me to create amazing frames for animation, or test different types of styles or combine them.
